lib/ssl/src/ssl_handshake.erl

```diff
@@ -3805,23 +3799,28 @@ path_validate([], _, _, _, _, _, _, _, _, _, {error, {bad_cert, root_cert_expire
 path_validate([], _, _, _, _, _, _, _, _, _, Error) ->
     Error;
 path_validate([{TrustedCert, Path} | Rest], ServerName, Role, CertDbHandle, CertDbRef, CRLDbHandle,
-              Version, SslOptions, CertExt, InvalidatedList, Error) ->
+              Version, SslOptions, StaplingInfo, InvalidatedList, Error) ->
```

Comment:
There can be more certificate extensions that the staplinginfo I do not understand why you are changing this variable name.

Reply:
There can be if we implement it!

Reply:
I think you're right. I was misleaded because StaplingInfo was in a lot of places.
For TLS1.3 in tls_handshake_1_3 all extensions will be collected delivered here.

I will change that.

Reply:
changed StaplingInfo into `ExtInfo` (picked it because map contains not only extensions, but also stapling state which is something additional)
18649553ea